osm2streets/src/transform/separate_cycletracks.rs

```diff
@@ -123,7 +123,7 @@ fn snap(streets: &mut StreetNetwork, input: Cycleway) {
         lt: LaneType::Buffer(BufferType::Planters),
         dir: Direction::Fwd,
         width: LaneSpec::typical_lane_width(LaneType::Buffer(BufferType::Planters)),
-        turn_restrictions: Vec::new(),
+        allowed_turns: Default::default(),
```

Comment:
If this gets tedious in the future, we could consider a `LaneSpec::new(LaneType, Direction)` that plugs in `typical_lane_width` does `default` for turns. Maybe when we get to per-lane stop lines or something, it'll be time